swyft/inference/ratios.py

```diff
@@ -99,22 +93,20 @@ def _init_networks(self, dataset):
     def train(
         self,
         dataset,
-        max_epochs: int = 10,
-        batch_size: int = 32,
-        lr_schedule: Sequence[float] = [1e-3, 3e-4, 1e-4],
-        early_stopping_patience: int = 1,
-        nworkers: int = 0,
-        percent_validation=0.1,
+        batch_size=64,
+        validation_size=0.1,
+        early_stopping_patience=10,
```

Comment:
You changed the default settings for batch_size and early_stopping_patience.  Even if I haven't done exhaustive hyper parameter searches, the parameters that I used as defaults have shown good performance.  In particular a small early_stopping_patience seems to be useful, probably since the networks that we train are very simple.  Unless you demonstrated somewhere that your new settings are better, please leave them as they were.  Since I'm not sure what the other settings do I cannot comment on them. But again: I am worried that these changes make things worse unless they were actually tested and compared with what we have.

Reply:
Is `early_stopping_patciance` actually still used?   How is its role relate to `reduce_lr_patience`?

Reply:
I guess I'm mostly worried about the following point:  Do your default settings generate now still the same output for our example notebooks or decrease their quality?

Reply:
I will check it later re: results.

We didn't really do early stopping because we trained again at a lower level after early stopping. That's why it worked.

See here for the new method:
https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.ReduceLROnPlateau.